services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int,) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall
+
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> None:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> str:
```

Comment:
The return type annotation for `get_movie_session_by_id` is `str`, but it should be `MovieSession` since the function returns a `MovieSession` object. Update the annotation to reflect the correct return type.